.travis.yml

```diff
@@ -30,6 +17,27 @@ before_install:
   - make install
   - cd ..
 
+script:
```

Comment:
Also, it runs coveralls for all versions of Node then, which is a little silly.

But, it *is* a lot more elegant to split it up, so I reverted this.